allure-generator/src/main/javascript/components/widgets-grid/WidgetsGridView.js

```diff
@@ -71,7 +71,8 @@ class WidgetsGridView extends View {
 
         this.addRegion(name, {el: el.find('.widget__body')});
        /// fetchAndShow(this, name, this.model, new Widget({model: this.model}))
-        this.getRegion(name).show(new Widget({model: this.model}));
+        const model = this.model;
+        this.getRegion(name).show(new Widget({model, name}));
```

Comment:
would be better to prepare the correct model here

```
new Widget({model: this.model.getWidgetData(name)})
```

instead of doing this inside the `Widget` constructor